_data-prepper/trace-tuning.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: Trace tuning
+nav_order: 12
+---
+
+# Trace tuning
+
+Data Prepper for trace analytics in version 0.8.x supports both vertical and horizontal scaling. Adjust the size of your single Data Prepper instance to meet your workload's demands and scale vertically. 
+
+<!--- Is there a better way to phrase the version part of this sentence?--->
+
+Scale horizontally by deploying multiple Data Prepper instances to form a cluster by using the [Core Peer Forwarder](https://github.com/opensearch-project/data-prepper/blob/main/docs/peer_forwarder.md). This enables Data Prepper instances to communicate with instances in the cluster, and is required for horizontally-scaling deployments.
+
+## Scaling tips
+
+The following sections describe useful tips for scaling. We recommend that you modify parameters based on the requirements. We also recommend that you monitor the Data Prepper host metrics and OpenSearch metrics to ensure the configuration is working as expected.
+
+### Buffer
+
+The total number of trace requests that Data Prepper is processing is equal to sum of `buffer_size` in `otel-trace-pipeline` and `raw-trace-pipeline`. The total number of trace requests to OpenSearch is equal to the product of `batch_size` and `workers` in `raw-trace-pipeline`.
```

Comment:
where does `raw-trace-pipeline` exist?

Reply:
This is referring to what is now called `raw-pipeline` in https://opensearch.org/docs/latest/data-prepper/pipelines/#trace-analytics-pipeline.